src/passes/OptimizeInstructions.cpp

```diff
@@ -422,6 +422,36 @@ struct OptimizeInstructions
           }
         }
       }
+      {
+        // -x * -y   ==>   x * y
```

Comment:
As this is just for integers, perhaps the comment could mention that, like `[integer]` somewhere maybe?

Reply:
sure!